w2/zadania/AntekGol/Zadanie7.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+
+public class Zadanie7 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int suma=0;
+        int liczb=0;
+        int n;
+        do {
+            System.out.print("Podaj liczbe wieksza od 100"+"\n");
+            n = scanner.nextInt();
+            liczb++;
+            suma+=n;
+
+        } while (n<=100);
+        System.out.print("Srednia podanych liczb to: "+suma/liczb);
```

Comment:
Średnia powinna być zmiennoprzecinkowa :D

Reply:
Jak poprawiłem zadanie 7 to co mam teraz zrobić, by dać merge do was?